testdata.log

```diff
@@ -8412,23 +8412,17 @@ SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 ) ORDER BY i;
 
 ---- 1349
 SELECT * FROM t WHERE  EXISTS ( SELECT * FROM t WHERE i == 2 );
```

Comment:
Thanks for the clarification. I think I also had this impression. 

The fix is to return `filterDefaultPlan` with `expr` set to `value{val: false}`

I can fix this in a separate PR.